Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -35,6 +36,7 @@ public class JsonEncoder : IJsonEncoder
         private static readonly char s_rightCurlyBrace = '}';
         private static readonly char s_leftSquareBracket = '[';
         private static readonly char s_rightSquareBracket = ']';
+        private ThreadLocal<StringBuilder> m_stringBuilderPool = new ThreadLocal<StringBuilder>(() => new StringBuilder());
```

Comment:
why do you need a ThreadLocal?

Reply:
there was concurrent failures in test at some point so had added this for debug. Reverted the change